components/org.wso2.carbon.status.dashboard.core/pom.xml

```diff
@@ -181,6 +181,35 @@
                     <basedir>${project.build.outputDirectory}/org/wso2/carbon/status/dashboard/core</basedir>
                 </configuration>
             </plugin>
+            <plugin><!-- Overridden from parent pom to exclude generated sources -->
```

Comment:
don't override. include the plugin within your component. refer a checkstyle enabled component